src/main/java/org/tugraz/sysds/runtime/instructions/fed/FEDInstructionUtils.java

```diff
@@ -29,14 +29,16 @@
 import org.tugraz.sysds.runtime.instructions.spark.WriteSPInstruction;
 
 public class FEDInstructionUtils {
+	// This is currently a rather simplistic to our solution of replacing instructions with their correct federated
```

Comment:
License 2020

Reply:
Oh btw. would be nice if the comments could be placed at the lines you are referring to (just a heads up, it is clear from the comments, so no big problem)